modules/installation/pages/ports.adoc

```diff
@@ -22,20 +22,65 @@ Opening these ports allows external network traffic to access the {productname}
 [cols="1,1,1,1", options="header"]
 .External Port Requirements for {productname} Server
 |===
-| Port number | Protocol | Used By | Notes
-| 22          |          |         | Required for ssh-push and ssh-push-tunnel contact methods.
-| 67          | TCP/UDP  | DHCP    | Required only if clients are requesting IP addresses from the server.
-| 69          | TCP/UDP  | TFTP    | Required if server is used as a PXE server for automated client installation.
-| 80          | TCP      | HTTP    | Required temporarily for some bootstrap repositories and automated installations.
+
+| Port number
+| Protocol
+| Used By
+| Notes
+
+| 22
+| -
+| -
+| Required for ssh-push and ssh-push-tunnel contact methods.
+
+| 67
+| TCP/UDP
+| DHCP
+| Required only if clients are requesting IP addresses from the server.
+
+| 69
+| TCP/UDP
+| TFTP
+| Required if server is used as a PXE server for automated client installation.
+
+| 80
+| TCP
+| HTTP
+| Required temporarily for some bootstrap repositories and automated installations.
 Port 80 is not used to serve the {webui}.
-| 443         | TCP      | HTTPS   | {webui}, client, and proxy requests.
-| 4505        |  TCP     | salt    | Required to accept communication requests from clients.
+
+| 443
+| TCP
+| HTTPS
+| {webui}, client, and proxy requests.
+
+| 4505
+| TCP
+| salt
+| Required to accept communication requests from clients.
 The client initiates the connection, and it stays open to receive commands from the Salt master.
-| 4506        | TCP      | salt    | Required to accept communication requests from clients.
+
+| 4506
+| TCP
+| salt
+| Required to accept communication requests from clients.
 The client initiates the connection, and it stays open to report results back to the Salt master.
-| 5222        | TCP      | osad    | Required to push OSAD actions to clients.
-| 5269        | TCP      | jabberd | Required to push actions to and from a proxy.
-| 25151       | TCP      | Cobbler |
+
+| 5222
+| TCP
+| osad
+| Required to push OSAD actions to clients.
+
+| 5269
+| TCP
+| jabberd
+| Required to push actions to and from a proxy.
+
+| 25151
+| TCP
+|Cobbler
+| -
+
```

Comment:
Please undo these changes

Reply:
These changes are fine. Tables syntax can be specified both ways, I was verifying that the adoc was not somehow broken. Rather the issue was that the translations were not complete for the tables. My bad for not checking the source first!

Reply:
No, these changes are not fine because you are breaking translations (e. g. Czech has already translated the tables).

Changing string breaks translations because translation memories are not as smart as humans, so they cannot identify a line break does not change meaning. It's OK to change strings to add, remove or update content; it's not OK to change strings just to change formatting.